tests/kani/Intrinsics/Sysconf/main.rs

```diff
@@ -0,0 +1,11 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+//
+//! Check support for `sysconf`.
+
+use libc;
+
+#[kani::proof]
+fn main() {
+    let page_size = unsafe { libc::sysconf(libc::_SC_PAGESIZE) } as usize;
```

Comment:
Question, how is sysconf modeled? Is there any check that we can ensure we get the correct result?

Reply:
I'm also curious about this... Do we get a nondet. `usize` value?

Reply:
This pull request depends on https://github.com/diffblue/cbmc/pull/7780. I'm still working on the model, which so far is just an over-approximation of the [spec](https://man7.org/linux/man-pages/man3/sysconf.3.html).